threadsd/main.go

```diff
@@ -37,6 +40,9 @@ func main() {
 	connGracePeriod := fs.Duration("connGracePeriod", time.Second*20, "Duration a new opened connection is not subject to pruning")
 	keepAliveInterval := fs.Duration("keepAliveInterval", time.Second*5, "Websocket keepalive interval (must be >= 1s)")
 	enableNetPubsub := fs.Bool("enableNetPubsub", false, "Enables thread networking over libp2p pubsub")
+	mongoUri := fs.String("mongoUri", "", "MongoDB URI (if not provided, an embedded Badger datastore will be used)")
+	mongoDatabase := fs.String("mongoDatabase", "threaddb", "MongoDB database name")
```

Comment:
I think is safer to not provide a default value, and error on startup if this isn't set explicitly through flag/env.
I fear that not providing a value might overlap with multiple instances running in the same cluster, which might be dangerous.

That's what I did in Powergate simply to avoid a misconfiguration (or simply doing nothing using defaults) that might cause a really bad situation.

Reply:
Cool, default removed